runtime/vm/createramclass.cpp

```diff
@@ -1577,6 +1577,41 @@ popLoadingOrLinkingStack(J9VMThread *vmThread, J9StackElement **stack, J9Pool *s
 	pool_removeElement(stackpool, topOfStack);
 }
 
+
+/**
+ * JEP 360: if super class/interface is sealed the inheriting subclass must be listed in the
+ * super's PermittedSubclasses attribute to be a legal subclass.
+ * @param superRomClass ROM class of super class or interface
+ * @param className name of subclass
+ * @param classNameLength length of subclass name
+ * @return TRUE if subclass can legally inherit the super, FALSE otherwise.
+ */
+static VMINLINE BOOLEAN
+isClassPermittedBySealedSuper(J9ROMClass *superRomClass, U_8* className, U_16 classNameLength)
+{
+	BOOLEAN result = FALSE;
+
+	if (! J9ROMCLASS_IS_SEALED(superRomClass)) {
```

Comment:
Once testing is complete, please remove the extra blank here and I'll merge.

Reply:
I have removed the space. The tests will not pass until AdoptOpenJDK/TKG#94 is merged.

Reply:
OK let me know when it's ready.